src/main/resources/schema.sql

```diff
@@ -126,24 +126,32 @@ CREATE OR REPLACE PROCEDURE tyomahdollisuus_data.import()
 BEGIN
   ATOMIC
 
-  INSERT INTO tyomahdollisuus(id)
-  SELECT id
-  FROM tyomahdollisuus_data.import;
+  INSERT INTO tyomahdollisuus(id, ammattiryhma)
+  SELECT i.id, jsonb(i.data -> 'ammattiryhma')
```

Comment:
This should probably be
```suggestion
  SELECT i.id, i.data ->> 'ammattiryhma'
```
(the `->>` operator extracts the given JSON object field as text)